charts/argo-cd/README.md

```diff
@@ -82,6 +82,9 @@ Changes in the `CustomResourceDefinition` resources shall be fixed easily by cop
 
 ## Upgrading
 
+### 4.3.*
+
+With this minor version, the notification notifier's `service.slack` is no longer configured by default. 
```

Comment:
Please add a newline between your text and the next heading to fulfill markdown lint. Also please add it to README.md.gotmpl

Reply:
updated